lib/api_browser/app/js/directives/type_label.js

```diff
@@ -27,12 +27,12 @@
       type: '='
     },
     link: function(scope, element) {
-
+      console.log(scope.type);
```

Comment:
BTW this is not in master yet IMHO. @blanquer can you remove this line from your branch please?